etc/hub.bash_completion.sh

```diff
@@ -8,13 +8,14 @@ fi
 
 # Check that git tab completion is available
 if declare -F _git > /dev/null; then
```

Comment:
This existing condition already wraps the entire logic of this file, right? How about if the only change we make is simply expand this condition to be:

```sh
if declare -F _git >/dev/null && ! declare -F __git_list_all_commands_without_hub >/dev/null; then
```

Checking if `__git_list_all_commands_without_hub` is defined means that this file already executed. What do you think?

Reply:
Sure, that seems like it'll work fine as well (and it avoids having to add a new global variable). I was originally trying to mimic the zsh completion script, just to get a bit more consistency. Made the change.